ASIM/dev/ASimTester/ASimTester.csv

```diff
@@ -1120,4 +1120,76 @@ ActorScope,string,Optional,ProcessEvent,,,
 TargetOriginalUserType,string,Optional,ProcessEvent,,,
 TargetScopeId,string,Optional,ProcessEvent,,,
 TargetScope,string,Optional,ProcessEvent,,,
-
+SrcIpAddr,string,Mandatory,Dhcp,IP Address,,
+IpAddr,string,Alias,Dhcp,IP Address,,SrcIpAddr
+RequestedIpAddr,string,Optional,Dhcp,IP Address,,
+SrcHostname,string,Mandatory,Dhcp,Hostname,,
+Hostname,string,Alias,Dhcp,,,DstHostname
+SrcDomain,string,Recommended,Dhcp,Domain,,
+SrcDomainType,string,Conditional,Dhcp,Enumerated,Windows|FQDN,SrcDomain
+SrcFQDN,string,Optional,Dhcp,FQDN,,
+SrcDvcId,string,Optional,Dhcp,,,
+SrcDvcScopeId,string,Optional,Dhcp,,,
+SrcDvcScope,string,Optional,Dhcp,,,
+SrcDvcIdType,string,Conditional,Dhcp,Enumerated,AzureResourceId|MDEid,SrcDvcId
+SrcDeviceType,string,Optional,Dhcp,Enumerated,Computer|Mobile Device|IOT Device|Other,
+SrcUserId,string,Optional,Dhcp,,,
+SrcUserIdType,string,Conditional,Dhcp,Enumerated,SID|UID|AADID|OktaId|AWSId,SrcUserId
+SrcUsername,string,Optional,Dhcp,,,
+Username,string,Alias,Dhcp,,,SrcUsername
+SrcUsernameType,string,Conditional,Dhcp,Enumerated,UPN|Windows|DN|Simple,SrcUsername
+SrcUserType,string,Optional,Dhcp,Enumerated,Regular|Machine|Admin|System|Application|Service Principal|Other,
+SrcOriginalUserType,string,Optional,Dhcp,,,
+SrcMacAddr,string,Mandatory,Dhcp,Mac Address,,
+DhcpLeaseDuration,int,Optional,Dhcp,,,
+DhcpSessionId,string,Optional,Dhcp,,,DhcpkSessionId
```

Comment:
DhcpSessionId alias value is incorrect and SessionId should have an Alias value of DhcpSessionId.

Reply:
Updated according to documentation:
https://learn.microsoft.com/en-us/azure/sentinel/normalization-schema-dhcp#:~:text=client%2C%20in%20seconds.-,DhcpSessionId,-Optional
https://learn.microsoft.com/en-us/azure/sentinel/normalization-schema-dhcp#:~:text=field.%0A%0AExample%3A%202099570186-,SessionId,-Alias